readme.md

```diff
@@ -54,30 +68,213 @@ Available environment variables:
 | HTTP_HOST | `0.0.0.0`     |                                                                                                           |
 | HTTP_PORT | `3000`        |                                                                                                           |
 
+### Routes
+
+| Method | Path                     | description                                                                         |
+| ------ | ------------------------ | ----------------------------------------------------------------------------------- |
+| POST   | /v1/decoder              | General decoder route. Need to specify [protocol](#protocols) in the requests body  |
+| POST   | /v1/decoder/analog       | Decoder for the `analog` protocol based devices                                     |
+| POST   | /v1/decoder/mtx          | Decoder for the `mtx` protocol based devices                                        |
+| POST   | /v1/decoder/mtxLora      | Decoder for the `mtxLora` protocol based devices                                    |
+| POST   | /v1/decoder/obisObserver | Decoder for the `obisObserver` protocol based devices                               |
+| POST   | /v1/encoder              | General encoder route. Need to specify [protocol](#protocols)  in the requests body |
```

Comment:
```suggestion
| POST   | /v1/encoder              | General encoder route. Need to specify [protocol](#protocols)  in the requests body. |
```

in case there are some full sentences here it would be nice to put dot at the end

and in other lines as well

Reply:
fixed